conformance/conformance_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package conformance
+package conformance_test
```

Comment:
can you put a comment why this package needs to be different name

Reply:
I will put a comment but the reason is to avoid a circular import